src/collectors/pe/data.cpp

```diff
@@ -123,6 +123,27 @@ bool PeData<_Bits>::convert_rva_to_offset(uint32_t rva,
     return false;
 }
 
+
+template <typename _Bits>
+bool PeData<_Bits>::convert_rva_to_offset64(uint64_t rva,
+    uint64_t &offset) const {
+
+    if (_sections.empty()){
```

Comment:
this test is useless, if _sections is empty, then the loop below is never run and it still returns false.